red_queen/games/applications/grovers.py

```diff
@@ -0,0 +1,244 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the MIT License.
```

Comment:
I'm like 80% sure we aren't using the MIT license, just the Apache, but maybe check with Matthew!
```suggestion
# Part of Red Queen Project.  This file is distributed under the Apache 2.0 License.
```

Reply:
Oh wait, you're right. I looked at an older file from this repository for reference and it had this. I'll make the change now. Thanks for catching it!